app/konnect/dev-portal/administrators/public-portal.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Public Developer Portal
+no_version: true
+---
+
+Enable public access to your Dev Portal. When Public Portal is enabled, developers
+don't need to register for the portal access and any new application registrations aren't
+allowed. The Register options are not available in the portal.
```

Comment:
```suggestion
Enable public access to your Dev Portal. With **Public Portal** enabled, anyone 
can access your Dev Portal without a login. Developers don't need to register 
for portal access and new application registrations aren't allowed. 

The Register options are not available in a public-facing portal.
```
Calling out that this disables portal login _in general_ - this was a common point of confusion from the bug bash right before launch, as an admin would try to access the portal and find themselves needing a specific dev portal account.